tasks/task_116_com2sense_commonsense_reasoning.json

```diff
@@ -0,0 +1,12051 @@
+    "Positive Examples": [
+        {
+            "input": "The glass fell of a three-story building, so it broke into pieces.",
+            "output": "True",
+            "explanation": "Falling down from a considerable height causes glasses to break, so this is a true sentence."
+        },
+        {
+            "input": "Marry was going out to work, so she asked her sixteen-year-old daughter to take care of her five-year-old son.",
+            "output": "True",
+            "explanation": "It makes sense to ask a teenage person to take care of their child sibling, so this is a true sentence."
+        },
+        {
+            "input": "Johnathan didn't have a hammer, so he used a cotton pad to drive the nail into the wood.",
+            "output": "False",
+            "explanation": "This sentence is not true, because cotton pads are soft and therefore can not be proper substitutes for hammers."
+        },
+        {
+            "input": "Suraya's best friend is getting married soon, so she will likely choose to go on a trip instead of helping her friend organize the ceremony.",
+            "output": "False",
+            "explanation": "This sentence is not true, because people tend to help their best friends when they have a big event, like a wedding, coming soon."
+        }
+    ],
+    "Negative Examples": [
+        {
+            "input": "For the twenty guests Sam was going to have over for dinner, she ordered two small pizzas.",
+            "output": "True",
+            "explanation": "Two small pizzas is not enough dinner for twenty people, so this sentence is not true."
+        },
+        {
+            "input": "Ali has decided to get up early from now on, so he is likely to go to bed later than before.",
+            "output": "True",
+            "explanation": "People are likely to go to bed earlier if they want to get up sooner, so this sentence is not true."
+        },
+        {
+            "input": "Jacob lives a 15-minute drive away from his office, so in order to be on-time for his 8 am meeting, he should leave home by 7:45.",
+            "output": "False",
+            "explanation": "Given the 15 minute driving time to a place, one should leave at least 15 minute before an event to catch it on-time, so this sentence is true."
+        },
+        {
+            "input": "My smart neighbour adviced me to use a paper tape to fix the hole in my bath tub.",
+            "output": "True",
+            "explanation": "Bath tubs are filled with water and a paper tape will dissolve in a bath tub, so a smart person would not give such an advice."
+        },
+        {
+            "input": "After a year of consistent practice, Jennifer was able to run a marathon.",
+            "output": "Correct",
+            "explanation": "The output should be either 'True' or 'False'."
+        }
+    ],
+    "Source": [
+        "Com2Sense"
```

Comment:
Expand this a bit, something like this: The instructions and the instance are extracted from the Com2Sense (<URL to the dataset>)